src/runtime/runtime_nrf52840.go

```diff
@@ -0,0 +1,149 @@
+//go:build nrf && nrf52840
```

Comment:
Seems like a lot of changes are needed for the nrf52840, how was this working before without all of this initialization?

Reply:
Until now, `machine.Serial.Configure` has only been called.
Except for nrf52840, `machine.Serial` was set to use `serial-uart.go`.
So we were able to use the same source code for all nrf's including nrf52840.

However, that does not allow us to merge the changes in #2631
In this case, nrf52840 wants to perform USB initialization with or without serial-uart.go.

https://github.com/tinygo-org/tinygo/blob/b65447c7d567eea495805656f45472cc3c483e03/src/runtime/runtime_nrf.go#L30-L34

Reply:
OK, understood. Thank you for the clarification.